app/views/layouts/_header.html.erb

```diff
@@ -7,15 +7,12 @@
   <%= stylesheet_link_tag "I18n" %>
 <% end %>
 
-<nav id="header" class="navbar navbar-inverse navbar-fixed-top">
-  <div class="container">
+<nav id="header" class="navbar navbar-expand-sm navbar-inverse bg-inverse">
```

Comment:
Here, navbar-expand-sm should have expanded the navbar desktop view. But the navbar remained collapsed.

Reply:
Looking... i see `Navbars are responsive by default, but you can easily modify them to change that. Responsive behavior depends on our Collapse JavaScript plugin.` -- do we need that plugin added?

https://getbootstrap.com/docs/4.1/components/navbar/

Reply:
yes maybe, it also says `.navbar-toggler for use with our collapse plugin and other navigation toggling behaviors.`

Reply:
is popper also needed? not sure for what... https://getbootstrap.com/docs/4.1/getting-started/introduction/#js

Reply:
https://getbootstrap.com/docs/4.1/components/collapse/#how-it-works but i can't see where the plugin is included...

Reply:
hmm but this says it's all compiled in: https://getbootstrap.com/docs/4.1/getting-started/javascript/#individual-or-compiled

Reply:
GOT IT! 

https://github.com/publiclab/plots2/blob/master/app/assets/stylesheets/style.css#L116-L154

Overrides over the years as we've tried to customize and tweak. Just be sure we respect the behaviors shown here, or something equivalent! One complaint with the previous navbar was that it was too eager to hide things as the screen got narrower. This gif shows exactly how it was solved; hopefully the new Bootstrap just makes this much easier!! 

https://github.com/publiclab/plots2/pull/3491